pkg/pillar/types/zedroutertypes.go

```diff
@@ -1870,6 +1871,25 @@ type NetworkXObjectConfig struct {
 type IpRange struct {
 	Start net.IP
 	End   net.IP
+	Size  int
+}
+
+// Contains used to eaavluate whether an IP address
+// is within the range
+func (ipRange IpRange) Contains(ipAddr net.IP) bool {
+	addr0 := ipRange.Start.To4()
+	addr1 := ipRange.End.To4()
+	addr := ipAddr.To4()
+	cur := uint(addr[3])<<24 + uint(addr[2])<<16 +
+		uint(addr[1])<<8 + uint(addr[0])
```

Comment:
Is this correct for different byte order?

The recommended way to compare seems to be in https://stackoverflow.com/questions/19882961/go-golang-check-ip-address-in-range

Reply:
Note that the addtoip and calculation of size use addr[0] << 24 i.e., reverse order. Hence makes sense to use the suggestion from stackoverflow.